config/crds/patches/kustomization.yaml

```diff
@@ -42,3 +42,10 @@ patchesJson6902:
       kind: CustomResourceDefinition
       name: kibanas.kibana.k8s.elastic.co
     path: apm-kibana-patches.yaml
+  # custom patches for EnterpriseSearch
```

Comment:
Proper name here is Enterprise Search, two words.

Reply:
This really refers to the `EnterpriseSearch CRD` (the technical implementation), I'd be in favor of keeping it as is. Also this is def not a user-facing file.